### DISCUSSION TITLE
customer/vendor RMA also payment allocations

### DISCUSSION THREAD

**Discussion by haniyasin** [blueseerERP/blueseer#147] [Q&A] [answered]
thanks for this great project i was woundering if there is a way to accomplish some of these tasks
customer/vendor RMA 
credit and debit notes
payment allocation (customer/vendor) invoices

**Comment by vaughnte** [comment, marked as answer]
BlueSeer does not have an explicit RMA solution at this time.   However, you can perform the function of RMA with several different steps.  For example, if your customer is returning a Finished Good (FG) for whatever reason, you have a couple of options depending on whether payment has been applied or not.    If the customer has not sent payment, then simply 'void' the invoice.   This will reverse the GL entries, reverse the shipper, restore the original order and reverse the inventory.    Then you simply place the inventory back into it's physical location.   This is the easiest option.   However, if the customer has already sent payment....and payment has been applied in the system, then you cannot void.   You will need to do an 'inventory adjustment' to receive back in the inventory (or scrap it) and a Credit Memo which will credit the Accounts Receivables effectively negating the original invoice.    If you are returning 'vendor' product, then you have but one option....1) Inventory Adjustment to remove vendor item from inventory...and 2)  reverse the Accounts Payable entries (if paid already) using 'Standard Transaction Maintenance'.
A 'nice to have' would be an RMA menu that 1) auto detects payment condition, 2) creates quality issue, 3) adjusts inventory all in one menu...but that's not in the works at the moment.   We may yet put it on the project board.

**Reply by haniyasin** [reply]
thanks for the detailed answer, i really apologize for my ignorance on the system i am trying hard to read and understand the documentation in pdf in the website :
1- The credit memo path seems great yet i searched the user manual in pdf for a clear mention to credit memos i could not (or i need to make it as a manual journal entry?) 
2- also i could not find a way to allocate payments whether for customer or vendor , or is it just like just a tag or flag to the invoice that it has been paid and need to refer to customer or vendor account balance,
